Makefile

```diff
@@ -25,7 +25,7 @@ export SHELLOPTS := errexit
 ROOT := github.com/kleveross/klever-model-registry
 
 # Target binaries. You can build multiple binaries for a single project.
-TARGETS := klever-model-registry klever-modeljob-operator
+TARGETS := model-registry modeljob-operator
```

Comment:
I remember that we decide to have one separate binary for inference?

Reply:
the inference binary is trtserver and openscoring jar package, not include source code.